### DISCUSSION TITLE
Installation Instructions for Fedora 40, Please.

### DISCUSSION THREAD

**Discussion by Y2hlZXNl** [microsoft/SysmonForLinux#178] [General]
Thank you for the great work on Sysmon for Linux! I have been using this tool and find it incredibly useful.

I noticed that the current installation instructions do not include steps for Fedora 40. It would be greatly beneficial for Fedora users if we could add specific instructions for this version.

Could you please consider updating the documentation to include the installation steps for Fedora 40? This would help ensure that users on the latest version of Fedora can easily set up and use Sysmon.

**Comment by MarioHewardt** [comment]
Hi - Thanks for reporting this. Could you file this as an Issue?
